test/EndToEnd.Tests/Utilities/SupportedNetCoreAppVersions.cs

```diff
@@ -3,10 +3,12 @@
 
 using System.Collections;
 
-namespace EndToEnd
+namespace EndToEnd.Tests.Utilities
 {
     public static class TargetFrameworkHelper
     {
+        public const string CurrentTfm = "net9.0";
```

Comment:
Could this instead come from ToolsetInfo.CurrentTargetFramework somehow?

Reply:
I didn't even know that was there! I'll update the PR to use that instead.

Reply:
Done.